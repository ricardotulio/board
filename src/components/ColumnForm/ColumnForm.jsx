import React from 'react'

const ColumnForm = ({ column, onChange, onSubmit }) => (
  <form onSubmit={ onSubmit }>
    <div>
      <label>
        Column Title:
        <input type='text' value={ column.title } onChange={ onChange } />
      </label>
      <button type="submit">Submit</button>
    </div>
  </form>
)

export default ColumnForm
