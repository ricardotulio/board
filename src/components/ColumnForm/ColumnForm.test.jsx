import {Â render } from '@testing-library/react'
import '@testing-library/jest-dom'
import ColumnForm from './ColumnForm'

it('should render a ColumnForm', () => {
  const { container } = render(<ColumnForm />)
  expect(container.firstChild).toMatchSnapshot()
})
