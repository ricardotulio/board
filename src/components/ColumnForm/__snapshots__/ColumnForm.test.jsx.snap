// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a ColumnForm 1`] = `
<form>
  <div>
    <label>
      Column Title:
      <input
        type="text"
      />
    </label>
  </div>
</form>
`;
